pkg/knuu/instance.go

```diff
@@ -258,11 +258,18 @@ func (i *Instance) AddFile(src string, dest string, chown string) error {
 	return nil
 }
 
+// AddFile adds a file to the instance
+// This function can only be called in the state 'Preparing'
+func (i *Instance) AddFile(src string, dest string, chown string) error {
+	// As dockers `ADD` works for both files and folders, we can just call Add here
+	return i.AddFile(src, dest, chown)
```

Comment:
Recursive call, should be:

```suggestion
	return i.Add(src, dest, chown)
```